TaskHouseUnitTests/WorkersControllerUnitTests.cs

```diff
@@ -0,0 +1,217 @@
+using System;
+using Xunit;
+using TaskHouseApi.Controllers;
+using TaskHouseApi.Repositories;
+using Microsoft.AspNetCore.Mvc;
+using TaskHouseApi.Model;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+
+namespace TaskHouseUnitTests
+{
+    public class WorkersControllerUnitTests
+    {
+        WorkersController controller;
+        IWorkerRepository repo;
+
+        public WorkersControllerUnitTests()
+        {
+            repo = new FakeWorkerRepository();
+            controller = new WorkersController(repo);
+        }
+
+        [Fact]
+        public async void WorkerController_Get_ReturnsObjectReponseWithCorrectEmpolyer_WhenGivenValidId()
+        {
+            Worker TestWorker = new Worker()
+            {
+                Id = 1,
+                Username = "1234",
+                Password = "+z490sXHo5u0qsSaxbBqEk9KsJtGqNhD8I8mVBdDJls=", //1234
+                Email = "test@test.com",
+                FirstName = "Bob1",
+                LastName = "Bobsen1",
+                Salt = "upYKQSsrlub5JAID61/6pA=="
+            };
+
+            var result = await controller.Get(TestWorker.Id);
+            var resultObjectResult = await controller.Get(TestWorker.Id) as ObjectResult;
```

Comment:
No need to call controller.Get again, just cast result, as so: 
var resultObjectResult = result as ObjectResult;

Reply:
#2c753e8